server/public/model/channel.go

```diff
@@ -35,6 +35,10 @@ const (
 
 	ChannelSortByUsername = "username"
 	ChannelSortByStatus   = "status"
+
+	ChannelReportingSortByDisplayName = "displayName"
```

Comment:
Should these not go into `report.go`?

Reply:
Actually, channel reporting stuff shouldn't even be in this PR. I've removed this now.